### PR TITLE
Functional tests - Add new test bulk actions languages

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add functional test `Bulk actions languages` (to be merged after #17287)
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/11_international/01_localization/languages/03_bulkActionsLanguages.js" URL_FO=Shop_URL npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17440)

### CODE REVIEW

File: tests/puppeteer/pages/BO/international/languages/index.js [hunk 1]
```diff
@@ -130,4 +139,59 @@ module.exports = class Languages extends LocalizationBasePage {
...
     return this.getTextContent(this.alertSuccessBlockParagraph);
   }
+
+  /* Bulk Actions Methods */
+  /**
+   * Enable / disable Suppliers by Bulk Actions
+   * @param enable
+   * @return {Promise<textContent>}
+   */
+  async bulkEditEnabledColumn(enable = true) {
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/international/languages/index.js]:
```suggestion
  async bulkEditEnabledColumn(isEnabled = true) {
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/puppeteer/pages/BO/international/languages/index.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/international/languages/index.js]: This function is not checking status, but changing it. Maybe naming param as `toEnable` is better, WDYT ?

File: tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/03_bulkActionsLanguages.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        await expect(textResult).to.to.contains(this.pageObjects.languagesPage.successfulCreationMessage);
+        const numberOfLanguagesAfterCreation = await this.pageObjects.languagesPage.getNumberOfElementInGrid();
+        await expect(numberOfLanguagesAfterCreation).to.be.equal(numberOfLanguages + index + 1);
+      });
+    });
+  });
+
+  describe('Enable, disable and delete with bulk actions', async () => {
+    const tests = [
+      {args: {action: 'disable', enabledValue: false}, expected: 'clear'},
```
Comment (PierreRambaud) [inline tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/03_bulkActionsLanguages.js]:
```suggestion
      {args: {action: 'disable', isEnabled: false}, expected: 'clear'},
```

File: tests/puppeteer/pages/BO/international/languages/index.js (hunk 1 above)
Comment (PierreRambaud) [reply tests/puppeteer/pages/BO/international/languages/index.js]: Agree with `toEnable`

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**